src/common/reactNativeProjectHelper.ts

```diff
@@ -84,4 +161,10 @@ export class ReactNativeProjectHelper {
         const haulVersion = packageJson.devDependencies && packageJson.devDependencies.haul;
         return !!haulVersion;
     }
+
+    private static getReactNativePackageVersionFromNodeModules(projectRoot: string, parsedPackageName: ParsedPackageName): Q.Promise<PackageVersion> {
```

Comment:
```suggestion
    private static getPackageVersionFromNodeModules(projectRoot: string, parsedPackageName: ParsedPackageName): Q.Promise<PackageVersion> {
```

Reply:
The `getReactNativePackageVersionFromNodeModules` function has been renamed to `getProcessedVersionFromNodeModules`, since `getPackageVersionFromNodeModules` is already used.